src/SAML2/Binding/HttpBindingInterface.php

```diff
@@ -55,6 +55,12 @@ public function receiveSignedAuthnRequest(Request $request);
 
     /**
      * @param Request $request
+     * @return \SAML2_AuthnRequest
+     */
+    public function receiveAuthnRequest(Request $request);
```

Comment:
I think this is a duplicate as there is already `receiveSignedAuthnRequest` and `receiveUnsignedAuthnRequest`